task_1/css_task/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Task - 1</title> 
+    <!--Local CSS-->
+    <link rel="stylesheet" href="./css/main.css">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.min.css">
+</head>
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-12 col-sm-12 col-md-12 col-lg-12 col-xl-12 d-flex justify-content-center">
+                <div class="task-one_form-layout">
+                    <div class="row task-one_form-input-spacing">   
+                        <div class="col-12 col-sm-12 col-md-6 col-lg-6 col-xl-6">
+                            <label for="firstName">First Name <span><i class="fas fa-asterisk task-one_form-required"></i></span></label>
+                            <input type="text" class="form-control task-one_form-input-name" id="user_name" placeholder="enter your first name" pattern="[a-zA-Z ][a-zA-Z ]+">
+                        </div>
+                        <div class="col-12 col-sm-12 col-md-6 col-lg-6 col-xl-6">
+                            <label for="lastName">Last Name <span><i class="fas fa-asterisk task-one_form-required"></i></span></label>
+                            <input type="text" class="form-control task-one_form-input-name" id="user_name" placeholder="enter your last name" pattern="[a-zA-Z ][a-zA-Z ]+">

```

Comment:
How is "pattern" being triggered?

Reply:
Here pattern takes in regex. I have used the basic that I know of.